resources/assets/js/components/Modal.tsx

```diff
@@ -161,22 +161,24 @@ Modal.Footer = function ModalFooter(props): React.ReactElement {
   return (
     <div data-c-padding="normal">
       <div data-c-grid="gutter middle">
-        {children.map(
-          (btn): React.ReactElement => {
-            const btnWithGridItemProp = React.cloneElement(btn, {
-              gridItem: children.length,
-            });
-            return btnWithGridItemProp;
-          },
-        )}
+        {Array.isArray(children) && children.length > 0
+          ? children.map(
+              (btn): React.ReactElement => {
+                const btnWithGridItemProp = React.cloneElement(btn, {
+                  gridItem: children.length,
```

Comment:
Seeing a warning in the console about `gridItem`

```
Warning: React does not recognize the `gridItem` prop on a DOM element. If you intentionally want it to appear in the DOM as a custom attribute, spell it as lowercase `griditem` instead. If you accidentally passed it from a parent component, remove it from the DOM element.
```

Reply:
Changed it to lowercase 👍